New log out flow (#135)
msupply-foundation/msupply_docs: content/docs/getting_started/tutorial.md

You will see the notification and the store name in the footer will be updated. From now, all actions will be in the newly selected store.

> Likewise, you can click on your username with the human icon:
> 
> ![Admin](/docs/getting_started/images/admin_button.png)
> 
> Upon clicking, it will show you a modal containing your information and the ability to log out:

Reviewer: NIT: ..slightly more concise? 

```suggestion
You can view your user information by clicking on your username:

![Admin](/docs/getting_started/images/admin_button.png)

You can also log out from here:
```
